src/aggregator/aggregation/options.go

```diff
@@ -28,14 +28,19 @@ import (
 )
 
 var (
-	defaultHasExpensiveAggregations = false
+	defaultEnableExpensiveAggregations               = false
+	defaultEnableAggregationLastValueAdjustTimestamp = false
```

Comment:
Wouldn't it make more sense for this to be true?

Reply:
For existing users of the aggregator with Graphite and native M3 metrics this would be a breaking change for them.

Best we can do I believe is to default to false and enable by default in the config.